client/src/App.js

```diff
@@ -82,26 +65,19 @@ function App() {
         } catch (error) {
             console.log(error)
         }
-    }, [data])
-
-    console.log(sections)
-    console.log(userProfile)
+    }, [contentVersions])
 
     return (
         <>
              {
                  loading || sections.length === 0 ? <Loader /> :
                     <div className="App">
-                        {/* <header className="App-header">
-                            <h1>Welcome to Gymkhana IITI</h1>
-                        </header> */}
-
                         <Router>
                             <Routes>
                             <Route path="/public" element={<Public />} />
                             <Route path="/admin" element={<Admin />} />
-                            <Route path="/admin/profile" element={<AdminProfilePage userProfile={userProfile[0]} updateGeneralDetails={updateGeneralDetails}/>} />
-                            <Route path="/admin/home" element={<AdminHomePage sections={sections} userProfile={userProfile[0]}/>} />
+                            <Route path="/admin/profile" element={<AdminProfilePage userProfile={userProfile[0]}/>} />
```

Comment:
why are we happing araay of user, it should be a single user always

Reply:
done